packages/wix-ui-tpa/src/components/DatePicker/DatePicker.st.css

```diff
@@ -192,6 +192,12 @@
     color: applyOpacity(value(CurrentNavigationArrowsColor), 0.7);
 }
 
+.calendar::header::yearAndMonthWrapper {
+    width: 12.6em;
```

Comment:
Maybe should be?
```suggestion
    width: 100%;
```

In responsiveness it looks like:
![image](https://user-images.githubusercontent.com/7141972/111070534-db2f9e00-84da-11eb-9ffb-95377aa307f1.png)

Which is different from the other examples (the arrows are not on the edge of left/right - not sure if okay or not...).

Also there is another concern - if a day in a different language is longer than `12.6em` it might overflow.

Reply:
Leigh from Booking checked what are the longest strings and found that it is “październik” and “Septiembre”.
The width of 12.6em was taken from a Calendar component that was developed on Events.

We checked it in different languages and also with year/month dropdowns and it looks good.

Regarding the width 100%, if we will set it to 100% the arrows would be aligned to the sides which doesn't look good in a large width.
![Screen Shot 2021-03-15 at 9 13 48](https://user-images.githubusercontent.com/66675260/111116628-f6e48400-856e-11eb-9a76-c89edaef0176.png)